@hubblecommerce/hubble/core/components/detail/ProductDetail.vue

```diff
@@ -0,0 +1,276 @@
+<template>
+    <div class="product-detail">
+         <div class="detail-top-wrp">
+            <div class="detail-left-wrp">
+                <product-detail-gallery v-if="product !== null" :media="product.media" :alt="product.name" />
+            </div>
+
+            <div class="buybox-wrp">
+                <product-detail-buybox v-if="product !== null" :data-product="product" />
+                <product-detail-service-info />
+
+                <tabs class="detail-tabs">
+                    <tab class="description-tab" :name="'Description'">
+                        <div v-if="product !== null" class="tab-content">
+                            <div v-text="product.description" />
+                        </div>
+                    </tab>
+                    <tab class="review-tab" :name="'Reviews'"> No reviews yet </tab>
+                </tabs>
+            </div>
+        </div>
+
+            <div class="detail-crosssellings">
+                <template v-if="loadCrosssellings">
+                    <lazy-product-detail-crosssellings :product-id="product.id" :cross-sellings="product.crossSellings" />
+                </template>
+            </div>
```

Comment:
Mind indents

Reply:
Fixed in [13cf94c](https://github.com/hubblecommerce/hubble-frontend-pwa/commit/13cf94c85400a618284e8af1b75adb1c035c9ab7)